lib/Airport_Challenge/weather.rb

```diff
@@ -0,0 +1,9 @@
+
+class Weather
+
+  def self.stormy?
+      different_weathers = [1,2,3,4,5]
+      different_weathers.sample == 1
+  end
```

Comment:
This is explicit about whats happening so its good code. If you wanted it shorter you could do:

`[1,2,3,4,5].sample == 1`